runtime-config-linux.md

```diff
@@ -77,93 +77,59 @@ There is a limit of 5 mappings which is the Linux kernel hard limit.
 
 ## Devices
 
-`devices` is an array specifying the list of devices to be created in the container.
+`devices` is an array specifying the list of devices that the MUST be available in the container.
```

Comment:
On Wed, Jan 20, 2016 at 10:15:41AM -0800, Vish Kannan wrote:
> > -`devices` is an array specifying the list of devices to be created in the container.
> > +`devices` is an array specifying the list of devices that the MUST be available in the container.
> 
> Can devices be setup using hooks?

Yeah.  Or with bind-mounts, or by putting the device nodes in the
rootfs (#98).  My attempt to summarize the pushback for dropping the
mknod section is in [1].

[1]: https://github.com/opencontainers/specs/pull/98#issuecomment-128804303